docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -198,6 +198,13 @@
               }
             }
           }
+        },
+        "exposedComponents": {
```

Comment:
Do you think it would be worth wrapping `exposedComponents` in an `extensions` object? Could make sense in case we'd like to add for example `exposedFunctions` or something in the future.

Reply:
I think that's a good idea, yeah! (More close to "open for extension but closed for modification")

Reply:
Addressed in https://github.com/grafana/grafana/pull/92732/commits/0a59b20aa04a4eafde14098e2395cb22f68c5a74.